bcda/cclf/cclf.go

```diff
@@ -152,26 +149,55 @@ func importCCLF0(ctx context.Context, fileMetadata *cclfFileMetadata) (map[strin
 }
 
 func importCCLF8(ctx context.Context, fileMetadata *cclfFileMetadata) (err error) {
-	db := database.Connection
-
-	repository := postgres.NewRepository(db)
-
 	if fileMetadata == nil {
 		err = errors.New("CCLF file not found")
 		fmt.Println(err.Error())
 		log.Error(err)
 		return err
 	}
 
+	db := database.Connection
+	repository := postgres.NewRepository(db)
+
+	e, err := repository.GetCCLFFileExistsByName(ctx, fileMetadata.name)
+	if err != nil {
+		fmt.Printf("failed to check existence of CCLF%d file.\n", fileMetadata.cclfNum)
```

Comment:
Something we can consider in the future is a [MultiWriter](https://golang.org/pkg/io/#MultiWriter). We can build our logger so it writes to a file as well as write to stdout. That way we don't have to remember to double log all of these entries.

Reply:
Great idea.